operators/bosh-operator/DirectorService.js

```diff
@@ -779,12 +772,6 @@ class DirectorService extends BaseDirectorService {
         })
       )
       .tap(() => logger.info('+-> Deleted service credentials'))
-      .tap(() => {
-        //TODO: Temporary fix. Binding info should always be fetched from etcd.
-        if (deploymentName === config.mongodb.deployment_name) {
```

Comment:
In current flow, there is no provision for deleting mongodb binding property??

Reply:
Yes, AFAIK I didn't see any unbind call for service-fabrik-mongodb.

Reply:
@jayeshjk you are right. There is no need for unbind of internal mongodb.